test/e2e/app-dir/ppr-full/ppr-full.test.ts

```diff
@@ -1,6 +1,60 @@
 import { createNextDescribe } from 'e2e-utils'
 
-const pages = [
+async function measure(stream: NodeJS.ReadableStream) {
+  let streamFirstChunk = 0
+  let streamEnd = 0
+
+  const data: {
+    chunk: string
+    emittedAt: number
+  }[] = []
+
+  await new Promise<void>((resolve, reject) => {
+    stream.on('data', (chunk: Buffer | string) => {
+      if (!streamFirstChunk) {
+        streamFirstChunk = Date.now()
+      }
```

Comment:
This is needed so we can verify if the static part was cached or not without requiring the header to be exposed on tests. This is consistent with @acdlite's suggestions on making the tests more "user-facing".